lib/i18n/tasks/command/options/data.rb

```diff
@@ -85,6 +85,8 @@ def print_forest(forest, opts, version = :show_tree)
             puts forest.inspect
           when 'keys'
             puts forest.key_names(root: true)
+          when 'key-values'
+            puts forest.key_values(root: true).map { |kv| kv.join(' ') }
```

Comment:
The key can contain spaces so perhaps it's better join with a `\t`?

Reply:
Good point, although wouldn't it make more sence to use a colon?

Reply:
A colon can also be part of the key name

Reply:
Okay, I'll make it a `\t`. Couldn't that also be a part of the key name though? As far as I can tell every character is allowed as long as it's properly escaped.

Reply:
True but `\t` is unlikely whereas I've seen `:` and ` ` in the bug reports here